toolkit/primitives/plutus-data/src/reserve.rs

```diff
@@ -38,26 +38,7 @@ pub struct ReserveStats {
 
 impl From<ReserveRedeemer> for PlutusData {
 	fn from(value: ReserveRedeemer) -> Self {
-		use ReserveRedeemer::*;
-		match value {
-			DepositToReserve { governance_version } => {
-				PlutusData::new_single_value_constr_plutus_data(
-					&BigNum::from(0_u64),
-					&PlutusData::new_integer(&BigInt::from(governance_version)),
-				)
-			},
-			ReleaseFromReserve => PlutusData::new_empty_constr_plutus_data(&BigNum::from(1_u64)),
-			UpdateReserve { governance_version } => {
-				PlutusData::new_single_value_constr_plutus_data(
-					&BigNum::from(2_u64),
-					&PlutusData::new_integer(&BigInt::from(governance_version)),
-				)
-			},
-			Handover { governance_version } => PlutusData::new_single_value_constr_plutus_data(
-				&BigNum::from(3_u64),
-				&PlutusData::new_integer(&BigInt::from(governance_version)),
-			),
-		}
+		PlutusData::new_empty_constr_plutus_data(&BigNum::from(value as u64))
```

Comment:
It should not pass. For example, `DepositToReserve` was `Constr 0 [1]` and now it is `Constr 0 []`.